src/metabase/lib/drill_thru/zoom_in_timeseries.cljc

```diff
@@ -69,11 +70,13 @@
                              (lib.temporal-bucket/temporal-bucket column)))]
            (assoc dimension :column-ref breakout))))
 
-(mu/defn- next-breakout-unit :- [:maybe ::lib.schema.temporal-bucketing/unit.date-time.truncate]
-  [column :- ::lib.schema.metadata/column]
-  (when-let [current-unit (lib.temporal-bucket/raw-temporal-bucket column)]
-    (when (contains? (set valid-current-units) current-unit)
-      (unit->next-unit current-unit))))
+(mu/defn- next-breakout-unit :- [:maybe ::lib.schema.drill-thru/drill-thru.zoom-in.timeseries.next-unit]
+  [query :- ::lib.schema/query
+   stage :- :int
+   field :- :mbql.clause/field]
```

Comment:
Is `:mbql.clause/field` the right type here? I'm assuming that the `(lib.util/clause-of-type? breakout :field)` in `matching-breakout-dimension` means we're guaranteed to get a `:mbql.clause/field` here.

Reply:
I'd use `column` everywehre

Reply:
Yes, I think that's correct here.